packages/wallets/coin98/src/adapter.ts

```diff
@@ -155,10 +155,46 @@ export class Coin98WalletAdapter extends BaseSignerWalletAdapter {
     }
 
     async signAllTransactions(transactions: Transaction[]): Promise<Transaction[]> {
-        const signedTransactions: Transaction[] = [];
-        for (const transaction of transactions) {
-            signedTransactions.push(await this.signTransaction(transaction));
+        try {
+            const wallet = this._wallet;
+            if (!wallet) throw new WalletNotConnectedError();
+
+            try {
+                const response = await wallet.request({ method: 'sol_signAllTransaction', params: [transactions] });
+                const publicKey = new PublicKey(response.publicKey);
+                const signatures = response.signature
```

Comment:
Actually, this too. Shouldn't this be `response.signatures`?

Reply:
Comment Fixed